cpp/src/parquet/column_reader.cc

```diff
@@ -1232,7 +1232,7 @@ int64_t TypedColumnReaderImpl<DType>::Skip(int64_t num_values_to_skip) {
       // Jump to the right offset in the Page
       int64_t values_read = 0;
       InitScratchForSkip();
-      ARROW_DCHECK_NE(this->scratch_for_skip_, nullptr);
```

Comment:
sorry for the churn, i guess if these macros already existed, then we should use the ARROW_ version to avoid conflicts.  I didn't realize they were both defined in the project.

Reply:
Done.